inc/priam/cluster.hpp

```diff
@@ -119,6 +119,12 @@ class cluster
         std::chrono::milliseconds delay,
         uint16_t                  max_executions) -> bool;
 
+     /**
+     * Controls the number of io threads for networking to the Cluster.
+     * @param io_threads Controls number of threads for network io
+     * @return True if threads were changed.
+     */
+    auto cluster::io_threads(uint16_t io_threads) -> bool;
```

Comment:
this `cluster::` shouldn't be here on the declaration

Reply:
Fixed erroneous copy/paste error.